src/routes/(site)/6/teams/team-section.svelte

```diff
@@ -51,6 +55,22 @@
     text-align: center;
   }
 
+  .dev-animation img {
+    animation-duration: 3s;
+    animation-name: spin;
+    animation-iteration-count: infinite;
+    animation-direction: normal;
+  }
+
+  @keyframes spin {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+
```

Comment:
Let's disable the animations in the case that a user prefers reduced motion.[^1]

```suggestion
@media (prefers-reduced-motion: reduce) {
  .dev-animation img {
    animation: none;
  }
}
```

> The setting is used to convey to the browser on the device that the user prefers an interface that removes, reduces, or replaces motion-based animations.
 Such animations can trigger discomfort for those with [vestibular motion disorders](https://www.a11yproject.com/posts/understanding-vestibular-disorders/).[^2]

[^1]: https://web.dev/articles/prefers-reduced-motion#working_with_the_media_query_from_css_and_javascript_contexts
[^2]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion